client/stylesheets/_modal.scss

```diff
@@ -372,6 +372,10 @@
 	margin: 0 !important;
 }
 
+.profileReceivedCardRight li a h3, .profileReceivedCardRight b {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Selector `profileReceivedCardRight` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line